__mocks__/electron.ts

```diff
@@ -1,3 +1,17 @@
 import { BrowserWindow } from './browser-window'
+import electron from 'electron'
+import log from 'electron-log'
 
+jest.mock('electron', () => ({
```

Comment:
We don't need to use `jest.mock` in here. Please refer https://github.com/electron/fiddle/blob/master/tests/mocks/electron.ts